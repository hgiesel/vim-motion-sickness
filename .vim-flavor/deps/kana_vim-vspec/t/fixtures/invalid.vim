describe 'Syntax highlighting'
  it 'does not work for command names in expectations'
    Expect before != after
    Expect context != describe
  end
  it 'does not work for operators not in expectations'
    not in a valid situation
  end
end
