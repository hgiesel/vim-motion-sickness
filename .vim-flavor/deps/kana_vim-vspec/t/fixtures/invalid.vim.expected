cccccccc SSSSSSSSSSSSSSSSSSSSS
  cc SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
    cccccc _______OO______
    cccccc ________OO_________
  ccc
  cc SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
    ... .. . ..... .........
  ccc
ccc
